src/commands/utility/ping.js

```diff
@@ -1,9 +1,10 @@
 const { SlashCommandBuilder } = require('discord.js');
+const { translateLanguage } = require('../../languages');
 
 module.exports = {
   data: new SlashCommandBuilder()
     .setName('ping')
-    .setDescription('Replies with Pong!'),
+    .setDescription(translateLanguage('es', 'pingCommand.description')),
```

Comment:
mejor que solo haya que pasarle la key, el language puede tomarlo de config.js internamente
```suggestion
    .setDescription(translateLanguage('pingCommand.description')),
```